client/src/pages/Bookings/EditButton.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Popover from '@material-ui/core/Popover';
+import Typography from '@material-ui/core/Typography';
+import SettingsIcon from '@material-ui/icons/Settings';
+import { updateStatus } from '../../helpers/APICalls/request';
+import Button from '@material-ui/core/Button';
+import Box from '@material-ui/core/Box';
+import { editButtonStyles } from './BookingStyles/EditButton';
+interface Props {
+  requestId?: string;
+  updateStatusState: (id: string, status: string, sectionName: string) => void;
+  sectionName: string;
+}
+
+export default function EditButton({ requestId, updateStatusState, sectionName }: Props): JSX.Element {
+  const pageClasses = editButtonStyles();
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const onClickHandler = async (status: string) => {
+    const { success } = await updateStatus(status, requestId);
+    if (success) {
+      const id = success['request']._id;
+      if (sectionName === 'nextBooking') updateStatusState(id, status, 'nextBooking');
+      else if (sectionName == 'current bookings') updateStatusState(id, status, 'currentBookings');
```

Comment:
Wouldn't it be better to have consistent section names here?

Reply:
I was using it differently in the their components, but I have changed it